bluebinder_wait.sh

```diff
@@ -4,6 +4,11 @@ while true
 do
     bt_status=$(/usr/bin/getprop init.svc.bluetooth-1-0)
     if [ "$bt_status" = "running" ] ; then
+#If bt address getted from outside script use it
```

Comment:
nitpick:
Can you indent this on the same line as the above line? with spaces
and maybe replace the comment with something like:
`# If bluetooth address is provided by another script`

Reply:
fixed